.pre-commit-hooks.yaml

```diff
@@ -51,3 +51,11 @@
   entry: requirements-txt-fixer
   language: python
   files: requirements.*\.txt$
+
+- id: google-java-code-format
```

Comment:
As an alternative: https://github.com/macisamuele/language-formatters-pre-commit-hooks/blob/master/language_formatters_pre_commit_hooks/pretty_format_java.py is also available (linked on https://pre-commit.com/hooks.html)

Reply:
From the doc:
"... This might be relevant for pretty-format-java and pretty-format-kotlin hooks. The hooks depends on having java on version 11 or greater installed on your machine. ..."

We have about 1/3 to 1/2 of our projects still using java8 I think.

We can switch the implementation on this repo if we wanted to without changing all the projects. For now the shell script seems a lot simpler and handles java8 and > java8 correctly.